src/chessground/Chessground.ts

```diff
@@ -161,6 +161,10 @@ export default class Chessground {
     anim(board.toggleOrientation, this)
   }
 
+  orienteWithColor = (color: Color): void => {
```

Comment:
I'd prefer not to have that change in chessground, it was fine the way you did it previously

Reply:
I do not know why would you prefer the code not to be centralized and modular depending on the functionality, meaning handling all the Chessground logic in the Chessground module.
But I do prefer it that way, I think I was doing it wrong previously.